doc/whats-new.rst

```diff
@@ -45,6 +45,9 @@ Deprecations
 Bug fixes
 ~~~~~~~~~
 
+- Plotting a pcolormesh with ``xscale="log"`` and/or ``yscale="log"`` works as
+  expected after improving the way the interval breaks are generated (:issue:`5333`).
+  By `Santiago Soler <https://github.com/santisoler>`_
```

Comment:
When you get the chance, please merge in the latest changes from git master and move this to the section for 0.18.3.